Docs/Practitioners/FindPractitioner.md

```diff
@@ -0,0 +1,72 @@
+
+
+# Find a Practitioner
+
+Retrieves federation ID
```

Comment:
Be more descriptive here. Finds a medical professional based upon the search criteria provided.

Also we're inconsistent in our terminology here. Medical professional and practitioner are the same thing. I believe we formally refer to them as medical practitioner so fix up all the places this isn't being used.